web/docs/language/basic-elements.md

```diff
@@ -365,9 +369,21 @@ export const signUp = async (args, context) => {
 ```
 
 :::info
-You don't need to worry about hashing the password yourself! Even when you are using Prisma's client directly and calling `create()` with a plain-text password, Wasp put middleware in place that takes care of hashing it before storing it to the database.
+You don't need to worry about hashing the password yourself! Even when you are using Prisma's client directly and calling `create()` with a plain-text password, Wasp put middleware in place that takes care of hashing it before storing it to the database. An additional middleware also performs field validation.
 :::
 
+To disable default validations, or add your own, you can do:
+```js
+const newUser = context.entities.User.create({
+        data: { email: 'some@email.com', password: 'this will be hashed!' },
+        _waspSkipDefaultValidations: true, // defaults to false
+        _waspCustomValidations: [
+          { name: 'password should not be password', fn: data => data.password !== 'password' },
```

Comment:
Maybe we can add quotes around "password"? When I first read it I thought it was some kind of a paradox test, I was ooh deep :D.
```suggestion
          { name: 'password should not be "password"', fn: data => data.password !== 'password' },
```

Reply:
Haha, great point/catch, I will wrap in quotes. Thanks!

Reply:
Haha, indeed :D!

Reply:
This has been addressed here: https://github.com/wasp-lang/wasp/pull/376/commits/e68484f528c5c939f1de4bf129672eabf363ec7f Thanks!